### PR TITLE
Update appsscript manifest to be more consistent

### PR DESCRIPTION
**Pull request by diminishedprime**

### CODE REVIEW

File: JSON-connect/src/appsscript.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "dataStudio": {
-    "name": "JSON-connect",
-    "logoUrl": "https://github.com/Lingewoud/JSON-community-connector/raw/master/assets/logo.png",
+    "name": "JSON Connect",
+    "logoUrl": "https://raw.githubusercontent.com/googledatastudio/community-connectors/master/JSON-connect/assets/logo.png",
     "company": "Lingewoud",
     "companyUrl": "https://lingewoud.nl/",
     "addonUrl": "https://github.com/Lingewoud/JSON-community-connector",
```
Comment (pfrisella) [inline JSON-connect/src/appsscript.json]: 'Learn more' (addonUrl) links should point to repo from which code is deployed, i.e. this repo

File: JSON-connect/src/appsscript.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-    "supportUrl": "https://github.com/Lingewoud/JSON-community-connector/issues",
+    "supportUrl": "https://github.com/googledatastudio/community-connectors/issues/new?title=JSON%20Connect%20Connector%20Issue",
     "shortDescription": "Free JSON connector. Fetch data from any JSON data source by URL. Use caching to speed up rendering.",
```
Comment (pfrisella) [inline JSON-connect/src/appsscript.json]: seems like content from shortDescription and description should be switched.

File: JSON-connect/src/appsscript.json [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
     "companyUrl": "https://lingewoud.nl/",
-    "addonUrl": "https://github.com/Lingewoud/JSON-community-connector",
-    "supportUrl": "https://github.com/Lingewoud/JSON-community-connector/issues",
-    "shortDescription": "Free JSON connector. Fetch data from any JSON data source by URL. Use caching to speed up rendering.",
-    "description": "Connect to JSON data by URL",
+    "addonUrl": "https://github.com/googledatastudio/community-connectors/blob/master/JSON-connect/README.md",
+    "supportUrl": "https://github.com/googledatastudio/community-connectors/issues/new?title=JSON%20Connect%20Connector%20Issue",
+    "shortDescription": "Connect to JSON data by URL",
+    "description": "Free JSON connector. Fetch data from any JSON data source by URL. Use caching to speed up rendering.",
     "sources": ["custom_json"],
```
Comment (pfrisella) [inline JSON-connect/src/appsscript.json]:
```suggestion
    "sources": ["CUSTOM_JSON"],
```

File: where-the-iss-at/src/appsscript.json [hunk 4]
```diff
@@ -4,8 +4,8 @@
     "logoUrl": "https://www.clickinsight.ca/wp-content/uploads/2019/01/cc-logo-orange-bg-400px.png",
     "company": "Google Data Studio Developer Relations",
     "companyUrl": "https://developers.google.com/datastudio/",
```
Comment (pfrisella) [inline where-the-iss-at/src/appsscript.json]:
```suggestion
    "companyUrl": "https://twitter.com/Marc_Soares",
```

File: bart-realtime-departures/src/appsscript.json [hunk 5]
```diff
@@ -4,8 +4,8 @@
     "logoUrl": "https://www.clickinsight.ca/wp-content/uploads/2019/01/cc-logo-orange-bg-400px.png",
     "company": "Google Data Studio Developer Relations",
```
Comment (pfrisella) [inline bart-realtime-departures/src/appsscript.json]:
```suggestion
    "company": "Marc Soares",
```

File: bart-realtime-departures/src/appsscript.json [hunk 6]
```diff
@@ -4,8 +4,8 @@
...
     "companyUrl": "https://developers.google.com/datastudio/",
```
Comment (pfrisella) [inline bart-realtime-departures/src/appsscript.json]:
```suggestion
    "companyUrl": "https://twitter.com/Marc_Soares",
```

File: firestore/src/appsscript.json [hunk 7]
```diff
@@ -12,10 +12,10 @@
   "dataStudio": {
     "name": "Google Cloud Firestore",
     "logoUrl": "http://i66.tinypic.com/2mevmeq.png",
-    "company": "Google",
+    "company": "Google Data Studio Developer Relations",
     "companyUrl": "https://firebase.google.com/products/firestore/",
```
Comment (pfrisella) [inline firestore/src/appsscript.json]: We shouldn't have identical company names point to different company URLs. So In this case either remove companyUrl or make consistent with others: https://developers.google.com/datastudio/ or change company name to reflect company URL

File: github/src/appsscript.json [hunk 8]
```diff
@@ -14,8 +14,8 @@
     "logoUrl": "https://raw.githubusercontent.com/googledatastudio/community-connectors/master/github/logo.png",
     "company": "Google Data Studio Developer Relations",
     "companyUrl": "https://developers.google.com/datastudio/",
-    "addonUrl": "https://github.com/googledatastudio/community-connectors/tree/master/github#readme",
-    "supportUrl": "https://github.com/googledatastudio/community-connectors/issues",
+    "addonUrl": "https://github.com/googledatastudio/community-connectors/blob/master/github/README.md",
+    "supportUrl": "https://github.com/googledatastudio/community-connectors/issues/new?title=GitHub%20Connector%20Issue",
     "description": "Get GitHub data for specific repositories.",
     "sources": ["GitHub"],
```
Comment (pfrisella) [inline github/src/appsscript.json]:
```suggestion
    "sources": ["GITHUB"],
```

File: google-fit/src/appsscript.json [hunk 9]
```diff
@@ -3,11 +3,11 @@
   "exceptionLogging": "STACKDRIVER",
   "dataStudio": {
     "name": "Google Fit",
-    "company": "Google",
+    "company": "Google Data Studio Developer Relations",
     "company_url": "https://fit.google.com/",
```
Comment (pfrisella) [inline google-fit/src/appsscript.json]: We shouldn't have identical company names point to different company URLs. So In this case either remove companyUrl or make consistent with others: https://developers.google.com/datastudio/ or change company name to reflect company URL.

File: meetup/src/appsscript.json [hunk 10]
```diff
@@ -14,8 +14,8 @@
     "logoUrl": "https://secure.meetupstatic.com/s/img/286374644891845767035/logo/meetup-logo-script-1200x630.png",
     "company": "Google Data Studio Developer Relations",
     "companyUrl": "https://developers.google.com/datastudio/",
-    "addonUrl": "https://github.com/googledatastudio/community-connectors/tree/master/meetup",
-    "supportUrl": "https://github.com/googledatastudio/community-connectors/issues",
+    "addonUrl": "https://github.com/googledatastudio/community-connectors/blob/master/meetup/README.md",
+    "supportUrl": "https://github.com/googledatastudio/community-connectors/issues/new?title=Meetup%20Connector%20Issue",
     "description": "Provides Meetup.com data for groups, pro groups, members, and events.",
     "sources": ["meetup"],
```
Comment (pfrisella) [inline meetup/src/appsscript.json]:
```suggestion
    "sources": ["MEETUP"],
```

File: nasa-apod/src/appsscript.json [hunk 11]
```diff
@@ -4,8 +4,8 @@
     "logoUrl": "https://www.clickinsight.ca/wp-content/uploads/2019/01/cc-logo-orange-bg-400px.png",
     "company": "Google Data Studio Developer Relations",
```
Comment (pfrisella) [inline nasa-apod/src/appsscript.json]:
```suggestion
    "company": "Marc Soares",
```

File: nasa-apod/src/appsscript.json [hunk 12]
```diff
@@ -4,8 +4,8 @@
...
     "companyUrl": "https://developers.google.com/datastudio/",
```
Comment (pfrisella) [inline nasa-apod/src/appsscript.json]:
```suggestion
    "companyUrl": "https://twitter.com/Marc_Soares",
```

File: pagespeed-insights/src/appsscript.json [hunk 13]
```diff
@@ -6,8 +6,8 @@
     "logoUrl": "https://www.gstatic.com/images/icons/material/product/2x/pagespeed_64dp.png",
     "company": "Google Data Studio Developer Relations",
```
Comment (pfrisella) [inline pagespeed-insights/src/appsscript.json]: Grant added this one, correct? If so...

```suggestion
    "company": "Grant Kemp",
```

File: pagespeed-insights/src/appsscript.json [hunk 14]
```diff
@@ -6,8 +6,8 @@
...
     "companyUrl": "https://developers.google.com/datastudio/",
```
Comment (pfrisella) [inline pagespeed-insights/src/appsscript.json]:
```suggestion
    "companyUrl": "https://twitter.com/ukdatageek",
```

File: slack/src/appsscript.json [hunk 15]
```diff
@@ -4,8 +4,8 @@
     "company": "Seo Gregory Pallichirayil",
     "companyUrl": "",
```
Comment (pfrisella) [inline slack/src/appsscript.json]: remove

File: slack/src/appsscript.json [hunk 16]
```diff
@@ -4,8 +4,8 @@
...
     "logoUrl": "https://www.gstatic.com/images/branding/product/1x/google_fonts_48dp.png",
-    "addonUrl": "https://api.slack.com/methods/channels.history",
-    "supportUrl": "https://developers.google.com/datastudio/connector/faq",
+    "addonUrl": "https://github.com/googledatastudio/community-connectors/blob/master/slack/README.md",
+    "supportUrl": "https://github.com/googledatastudio/community-connectors/issues/new?title=Slack%20Connector%20Issue",
     "description": "This connector can connect to your any specified Slack channel, and bring the data to the Data Studio.",
```
Comment (pfrisella) [inline slack/src/appsscript.json]:
```suggestion
    "description": "Connect to any Slack channel and bring that data into Data Studio.",
```

File: spotify/src/appsscript.json [hunk 17]
```diff
@@ -13,10 +13,10 @@
...
-    "company": "Google Developer Relations",
+    "company": "Google Data Studio Developer Relations",
     "companyUrl": "https://developers.google.com/datastudio/",
-    "addonUrl": "https://github.com/googledatastudio/community-connectors/tree/master/spotify-recent-plays",
-    "supportUrl": "https://github.com/google/datastudio/issues",
+    "addonUrl": "https://github.com/googledatastudio/community-connectors/blob/master/spotify/README.md",
+    "supportUrl": "https://github.com/googledatastudio/community-connectors/issues/new?title=Spotify%20Connector%20Issue",
     "description": "This connector queries the most popular tracks for an artist through the Spotify API.",
     "shortDescription": "Query track popularity statistics by artist through Spotify.",
     "sources": ["Spotify"]
```
Comment (pfrisella) [inline spotify/src/appsscript.json]:
```suggestion
    "sources": ["SPOTIFY"]
```

File: stackoverflow-questions/src/appsscript.json [hunk 18]
```diff
@@ -4,8 +4,8 @@
     "logoUrl": "https://cdn.sstatic.net/Sites/stackoverflow/company/img/logos/se/se-icon.png",
     "company": "Google Data Studio Developer Relations",
     "companyUrl": "https://developers.google.com/datastudio/",
-    "addonUrl": "https://github.com/googledatastudio/community-connectors/tree/master/stackoverflow-questions",
-    "supportUrl": "https://github.com/googledatastudio/community-connectors/issues",
+    "addonUrl": "https://github.com/googledatastudio/community-connectors/blob/master/stackoverflow-questions/README.md",
+    "supportUrl": "https://github.com/googledatastudio/community-connectors/issues/new?title=Stack%20Overflow%20Questions%20Connector%20Issue",
     "description": "Get Stack Overflow questions and relevant metadata by tag.",
     "sources": ["Stack Overflow"],
```
Comment (pfrisella) [inline stackoverflow-questions/src/appsscript.json]:
```suggestion
    "sources": ["STACK_OVERFLOW"],
```

File: star-wars-api/src/appsscript.json [hunk 19]
```diff
@@ -4,8 +4,8 @@
     "logoUrl": "https://pbs.twimg.com/profile_images/464164067360247808/Co7ITB9q_400x400.png",
     "company": "Google Data Studio Developer Relations",
```
Comment (pfrisella) [inline star-wars-api/src/appsscript.json]:
```suggestion
    "company": "Marc Soares",
```

File: star-wars-api/src/appsscript.json [hunk 20]
```diff
@@ -4,8 +4,8 @@
...
     "companyUrl": "https://developers.google.com/datastudio/",
```
Comment (pfrisella) [inline star-wars-api/src/appsscript.json]:
```suggestion
    "companyUrl": "https://twitter.com/Marc_Soares",
```

File: strava-activities/src/appsscript.json [hunk 21]
```diff
@@ -15,8 +15,8 @@
     "logoUrl": "https://1n4rcn88bk4ziht713dla5ub-wpengine.netdna-ssl.com/wp-content/uploads/2017/12/press-thumbnail-asset-logo-02.jpg",
     "company": "Google Data Studio Developer Relations",
     "companyUrl": "https://developers.google.com/datastudio/",
-    "addonUrl": "https://github.com/googledatastudio/community-connectors/tree/master/strava-activities",
-    "supportUrl": "https://github.com/googledatastudio/community-connectors/issues",
+    "addonUrl": "https://github.com/googledatastudio/community-connectors/blob/master/strava-activities/README.md",
+    "supportUrl": "https://github.com/googledatastudio/community-connectors/issues/new?title=Strava%20Activities%20Connector%20Issue",
     "description": "Get strava activity data.",
     "sources": ["strava"]
```
Comment (pfrisella) [inline strava-activities/src/appsscript.json]:
```suggestion
    "sources": ["STRAVA"]
```

File: where-the-iss-at/src/appsscript.json [hunk 22]
```diff
@@ -4,8 +4,8 @@
     "logoUrl": "https://www.clickinsight.ca/wp-content/uploads/2019/01/cc-logo-orange-bg-400px.png",
     "company": "Google Data Studio Developer Relations",
```
Comment (pfrisella) [inline where-the-iss-at/src/appsscript.json]:
```suggestion
    "company": "Marc Soares",
```

File: senuto/src/appsscript.json [hunk 23]
```diff
@@ -7,8 +7,8 @@
     "company": "Senuto",
     "companyUrl": "http://senuto.com/",
     "logoUrl": "https://images.senuto.com/img/logo_senuto_small.png",
-    "addonUrl": "http://senuto.com/",
-    "supportUrl": "http://senuto.com/",
+    "addonUrl": "https://github.com/googledatastudio/community-connectors/blob/master/senuto/README.md",
+    "supportUrl": "https://github.com/googledatastudio/community-connectors/issues/new?title=Senuto%20Connector%20Issue",
     "description": "SENUTO Community Connector for Data Studio"
```
Comment (pfrisella) [inline senuto/src/appsscript.json]:
```suggestion
    "description": "SENUTO Community Connector for Data Studio."
```

**Comment by diminishedprime** [comment]
@googlebot - can you try again.

**Pull request merged**